FeedSX/screens/postDetail/index.tsx

```diff
@@ -494,17 +560,30 @@ const PostDetail = (props: IProps) => {
 
   // this function calls the edit comment api
   const commentEdit = async () => {
+    // convert the mentions to route
+    const convertedEditedComment = replaceMentionValues(
+      commentToAdd,
+      ({id, name}) => {
+        const PATH = extractPathfromRouteQuery(id);
+        if (!PATH) {
+          return `<<${name}|route://${name}>>`;
+        } else {
+          return `<<${name}|route://${id}>>`;
+        }
+      },
```

Comment:
move this function to a util, as this duplicate function called in various places